Assets/BossRoom/Scripts/Client/Game/State/ClientMainMenuState.cs

```diff
@@ -46,6 +45,15 @@ protected override void InitializeScope()
         [Inject]
         async void InjectDependenciesAndInitialize(AuthenticationServiceFacade authServiceFacade, LocalLobbyUser localUser, LocalLobby localLobby)
         {
+            #if UNITY_EDITOR
+            if (string.IsNullOrEmpty(UnityEditor.CloudProjectSettings.projectId))
```

Comment:
There's no way to check with Application.cloudProjectId? And this way it would work in builds too?